src/geode/geosciences/gocad_common.cpp

```diff
@@ -399,6 +399,108 @@ namespace geode
             file << "END_ORIGINAL_COORDINATE_SYSTEM" << EOL;
         }
 
+        PropHeaderData read_prop_header( std::ifstream& file )
+        {
+            PropHeaderData header;
+            auto opt_line =
+                geode::detail::goto_keyword_if_it_exists( file, "PROPERTIES" );
+            if( !opt_line )
+            {
+                geode::Logger::info(
+                    "Token PROPERTIES could not be found in the file, "
+                    "attributes will not be loaded." );
+                return header;
```

Comment:
Is this case tested? I am unsure about the state of the ifstream after the goto_keywords call.

Reply:
Yes it is tested in the test_lso.cpp, where a lso file is imported, exported without attributes, and re-imported.
The goto_keyword_if_it_exists function returns the ifstream to its beginning if the keyword is not found